File: Learn C#/20230725/MyBankApp/Models/Transaction.cs
using MvcMovie.Models;
using System;
using System.Collections.Generic;
using System.Data.Entity;
using System.Web.Security;


namespace MyBankApp.Models
{
    public class Transaction
    {
        public int ID { get; set; } // トランザクションのID
        public decimal Amount { get; set; } // 金額
        public DateTime Date { get; set; } // 日付
        public string Note { get; set; } // 内容

        public Transaction(decimal amount, DateTime date, string note)
        {
            Amount = amount;
            Date = date;
            Note = note;
        }
    }
}

File: Learn C#/20230728/MyBankAppvb/Models/Transaction.vb
Imports System
Imports System.Collections.Generic
Imports System.Data.Entity
Imports System.Web.Security

Namespace MyBankAppvb.Models
    Public Class Transaction
        Public Property ID As Integer ' トランザクションのID
        Public Property Amount As Decimal ' 金額
        Public Property [Date] As DateTime ' 日付
        Public Property Note As String ' 内容

        Public Sub New(ByVal amount As Decimal, ByVal [date] As DateTime, ByVal note As String)
            amount = amount
            [date] = [date]
            note = note
        End Sub
    End Class
End Namespace
